corehq/apps/toggle_ui/views.py

```diff
@@ -332,10 +332,8 @@ def _get_service_type(toggle):
     for enabled in toggle.enabled_users:
         name = _enabled_item_name(enabled)
         if _namespace_domain(enabled):
-            subscription = Subscription.get_active_subscription_by_domain(name)
-            if subscription:
-                service_type[name] = subscription.service_type
-    return service_type
+            service_type, plan = get_subscription_info(name)
+    return f"{service_type} : {plan}"
```

Comment:
previously this was returning a dict, but now it's returning a string.

Reply:
Fixed